src/index.js

```diff
@@ -1,21 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+
+
 import './index.css';
 import App from './components/App';
-import * as serviceWorker from './serviceWorker';
```

Comment:
We can delete this file then

Reply:
Ok, I am not sure if we need this in the future. I did not delete it.